### DISCUSSION TITLE
Make znap install install in order and ignore completion that have install

### DISCUSSION THREAD

**Discussion by thanh125643** [marlonrichert/zsh-snap#113] [Q&A] [answered]
When i use znap install like this:
```zsh
znap install ohmyzsh/ohmyzsh zsh-users/zsh-completions
```
this will keep symbolic file from them on one completion each time zsh load. Here exmaple:
```zsh
 ~/.local/share/zsh/site-functions/_artisan -> ~[zsh-completions]/src/_artisan
~/.local/share/zsh/site-functions/_coffee -> ~[zsh-completions]/src/_coffee
~/.local/share/zsh/site-functions/_fab -> ~[zsh-completions]/src/_fab
```
Can you make this like if you symbolic ohmyzsh/ohmyzsh have that completion ,you dont load another from zsh-users/zsh-completions
Sorry my bad english

**Comment by marlonrichert** [comment, marked as answer]
That's not how you're supposed to use `znap install`. 🙂  You're supposed to run it just once on the command line and that's it. No need to add anything to your `.zshrc` file.

**Reply by thanh125643** [reply]
Like i want it auto for me when i move to new computer

**Reply by thanh125643** [reply]
And also i think you will need some explain on your readme file for this znap install not use in .zshrc .

**Reply by marlonrichert** [reply]
True, I could make that more clear.

**Reply by marlonrichert** [reply]
I updated the [Readme](https://github.com/marlonrichert/zsh-snap/blob/main/README.md). Hopefully, it's clearer now.
